### PR TITLE
add passwordless and start distributed all in one scripts

### PR DESCRIPTION
**Pull request by glorysdj**
This change is [Reviewable](https://reviewable.io/reviews/intel-analytics/analytics-zoo/3613)

### CODE REVIEW

File: ppml/trusted-big-data-ml/scala/docker-graphene/start-distributed-big-data-ml.sh [hunk 1]
```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+
+#set -x
+
+source ./environment.sh
+
+echo "### phase.1 distribute the keys and password and data"
+echo ">>> $MASTER"
+ssh root@$MASTER "rm -rf $KEYS_PATH && rm -rf $SECURE_PASSWORD_PATH && rm -rf $DATA_PATH && mkdir -p $AZ_PPML_PATH"
```
Comment (qiuxin2012) [inline ppml/trusted-big-data-ml/scala/docker-graphene/start-distributed-big-data-ml.sh:9]: so master should have no passwd ssh to all workers? Have you document this?

Comment (glorysdj) [reply ppml/trusted-big-data-ml/scala/docker-graphene/start-distributed-big-data-ml.sh:9]: yes, in README
##### setup passwordless ssh login to all the nodes.

**Review by qiuxin2012 — APPROVED** [review]

File: ppml/trusted-big-data-ml/scala/docker-graphene/start-distributed-big-data-ml.sh (hunk 1 above)
Comment (jason-dai) [reply ppml/trusted-big-data-ml/scala/docker-graphene/start-distributed-big-data-ml.sh:9]: > yes, in README
> 
> ##### setup passwordless ssh login to all the nodes.

While this is OK for current implementation, is it possible to do this for K8s deployment in future?

Comment (glorysdj) [reply ppml/trusted-big-data-ml/scala/docker-graphene/start-distributed-big-data-ml.sh:9]: > > yes, in README
> > ##### setup passwordless ssh login to all the nodes.
> 
> While this is OK for current implementation, is it possible to do this for K8s deployment in future?

no, in k8s, we may need another config file like k8s service or helm charts to organize and deploy all the components.
we can try ppml on graphene on k8s later after this release.

**Pull request merged**